code/modules/mob/living/carbon/human/species_types/polysmorphs.dm

```diff
@@ -39,3 +39,14 @@
 	var/randname = polysmorph_name()
 
 	return randname
+
+/datum/species/polysmorph/on_species_gain(mob/living/carbon/C, datum/species/old_species, pref_load)
+	.=..()
+	var/mob/living/carbon/human/H = C
+	if(H.physiology)
+		H.physiology.armor.wound += 10	//Pseudo-exoskeleton makes them harder to wound
+
+/datum/species/polysmorph/on_species_loss(mob/living/carbon/human/C, datum/species/new_species, pref_load)
+	.=..()
+	if(C.physiology)
+		C.physiology.armor.wound -= 10
```

Comment:
```suggestion
/datum/species/polysmorph/on_species_gain(mob/living/carbon/human/H, datum/species/old_species, pref_load)
	. = ..()
	H?.physiology?.armor?.wound += 10	//Pseudo-exoskeleton makes them harder to wound

/datum/species/polysmorph/on_species_loss(mob/living/carbon/human/H, datum/species/new_species, pref_load)
	. = ..()
		H?.physiology?.armor?.wound -= 10
```

Reply:
I thought we learned our lesson with using conditionals like this with the genetics neuroplasticity stun immunity thing

Reply:
Also on_species_gain specifies just carbons, not carbon/humans like on_speies_loss does